project/src/main.cpp

```diff
@@ -1 +1,22 @@
-// Write code here
\ No newline at end of file
+#include<iostream>
+using namespace std;
+    class company{
```

Comment:
Class names should start with Capital Letters

Reply:
ok